lib/ansible/modules/system/java_cert.py

```diff
@@ -295,24 +322,33 @@ def main():
 
     module = AnsibleModule(
         argument_spec=argument_spec,
-        required_one_of=[['cert_path', 'cert_url', 'pkcs12_path']],
+        required_one_of=[['cert_path', 'cert_content', 'cert_url', 'pkcs12_path']],
         required_together=[['keystore_path', 'keystore_pass']],
         mutually_exclusive=[
-            ['cert_url', 'cert_path', 'pkcs12_path']
+            ['cert_url', 'cert_path', 'cert_content', 'pkcs12_path']
         ],
         supports_check_mode=True,
+        add_file_common_args=True,
     )
+    # Force keytool message in english
+    module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C', LC_CTYPE='C')
 
     url = module.params.get('cert_url')
     path = module.params.get('cert_path')
     port = module.params.get('cert_port')
+    content = module.params.get('cert_content')
+    cert_alias = module.params.get('cert_alias') or url
+
+    if content is not None:
+        if not cert_alias:
+            module.fail_json(changed=False,
+                             msg="Using content import from requires alias argument.")
+        path = "%s/%s" % (os.path.dirname(os.path.realpath(__file__)), cert_alias)
```

Comment:
`path` isn't created and is used with `-file` switch in `import_cert_path` method but `keytool` manpage states that the `value is expected to be the fully qualified path of a file accessible by the File System`: I am not sure how do that work ?